CMake/test_labels.cmake

```diff
@@ -1,18 +1,20 @@
 function(ADD_TEST_LABELS TEST_NAME TEST_LABELS)
-  set(SUCCESS FALSE)
   set(TEST_LABELS_TEMP "")
-  execute_process(
-    COMMAND ${qmcpack_SOURCE_DIR}/tests/scripts/test_labels.py ${TEST_NAME} ${QMC_CUDA} ${QMC_COMPLEX}
-            ${QMC_MIXED_PRECISION}
-    OUTPUT_VARIABLE TEST_LABELS_TEMP
-    RESULT_VARIABLE SUCCESS)
-  #MESSAGE("  Label script return value: ${SUCCESS}")
-  if(NOT ${SUCCESS} STREQUAL "0")
-    message("Warning: test labeling failed.  Test labeling error output:\n${TEST_LABELS_TEMP}")
-    set(TEST_LABELS_TEMP "")
-    #ELSE()
-    #  MESSAGE("  Test: ${TEST_NAME}")
-    #  MESSAGE("    ${TEST_LABELS_TEMP}")
+  if (DEFINED ${TEST_NAME}_${QMC_CUDA}_${QMC_COMPLEX}_${QMC_MIXED_PRECISION})
```

Comment:
`NOT DEFINED`?
Also please start with `TEST_LABELS_`
So `TEST_LABELS_${TEST_NAME}_${QMC_CUDA}_${QMC_COMPLEX}_${QMC_MIXED_PRECISION}`